Pod/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -19,30 +19,38 @@ public class AVFoundationPlayback: Playback {
         return "AVPlayback"
     }
 
-    public override var selectedSubtitle: AVMediaSelectionOption? {
+    public override var selectedSubtitle: Subtitle? {
         get {
-            return getSelectedMediaOptionWithCharacteristic(AVMediaCharacteristicLegible)
+            let option = getSelectedMediaOptionWithCharacteristic(AVMediaCharacteristicLegible)
+            return Subtitle.fromAVMediaSelectionOption(option)
         }
         set {
-            setMediaSelectionOption(newValue, characteristic: AVMediaCharacteristicLegible)
+            if let newOption = newValue?.raw as? AVMediaSelectionOption {
```

Comment:
There should be a way to disable subtitles (setting the `AVMediaSelectionOption` to `nil`)